tests/integration/self_update_test.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+include './src/lib/kwio.sh'
+include './tests/unit/utils.sh'
+include './tests/integration/utils.sh'
+
+declare -g expected_output
+
+function self_update_test_helper()
+{
+  local kw_branch="$1"
+  local tmp_kw_dir
+  local repo_url='https://github.com/kworkflow/kworkflow'
+  local base_version
+  local distro
+  local container
+  local output
+  local expected_output
+  local head_commit_hash
+
+  tmp_kw_dir=$(mktemp --directory)
```

Comment:
I'm kind of confused: shouldn't this command be run inside the container? I think the temporary directory is being created on the host machine and that the test doesn't fail because the path returned will be something like `/tmp/tmp.XXXXXX`, which is a valid path for the container. Confirm if this is the case and, if so, run this command inside each container with `container_exec`.

Reply:
Hi @davidbtadokoro.

I hadn't noticed that, it's been fixed now. Thank you :-)